### DISCUSSION TITLE
Need help on correcting the spelling mistake in the URL ( my last name is typo)

### DISCUSSION THREAD

**Discussion by pravipan** [knowledgefutures/pubpub#3208] [Support]
How to get name typo corrected?

**Comment by eastofesten** [comment]
Hi, @pravipan - happy to help! Can you share the information for the community & URL you're working in? (You can also email help@pubpub.org if you prefer.)

**Comment by pravipan** [comment]
Appreciate your response.

Here is the URL --- https://ijgis.pubpub.org/user/pravin-*pansey *

My last name is "*Pandey*".
Please help me correct "Pansey" to "*Pandey*"
